examples/nats-echo/main.go

```diff
@@ -115,6 +115,14 @@ func main() {
 				nc.Publish(msg.Reply, msg.Data)
 			}
 		}
+	}
+
+	nc.QueueSubscribe(subj, "echo", func(msg *nats.Msg) {
+		handleMsg(msg)
+	})
+	allSubj := subj + ".all"
```

Comment:
Let's make this status.

Reply:
And for this one we just should send back our geo I think? Maybe if you want add in an identifier at startup to customize?

Reply:
Fixed; it's a status and there is an identifier now.